github-actions/scan/src/post-scan.ts

```diff
@@ -69,3 +55,180 @@ export async function uploadArtifact(name: string, paths: string[]) {
     }
     core.endGroup();
 }
+
+/**
+ * Reads the given field from the SecHub JSON report.
+ * @param {string} field - The field relative to root, where the value should be found. The field can be a nested field, e.g. result.count.
+ * @returns {*} - The value found for the given field or undefined if not found.
+ */
+function getFieldFromJsonReport(field: string): any {
+    // Construct the full file path of the JSON report
```

Comment:
The method `getFieldFromJsonReport` is used 4 times by a scan:

- jobUUID
- trafficLight
- result.count
- result.findings

Every time the complete json data is read from filesystem and parsed as a new JSON object.

What about changing
- `getFieldFromJsonReport(field: string): any` to something like
  `getFieldFromJsonReport(field: string, json: JSON): any`
- and handle the JSON read only one time + use the JSON object as parameter 


_Topic: reduce JSON reads/objects_